pkg/scripts/kubeadm.go

```diff
@@ -67,7 +67,7 @@ var (
 
 	kubeadmPauseImageVersionScriptTemplate = heredoc.Doc(`
 		sudo kubeadm config images list --kubernetes-version={{ .KUBERNETES_VERSION }} |
-			grep "k8s.gcr.io/pause" |
+			grep "registry.k8s.io/pause" |
```

Comment:
This script have to handle both cases, for older k8s versions will still return k8s.gcr.io.

Reply:
@kron4eg I just pushed a commit to fix this.

Reply:
The problem is: at the time when this script run, the config file is not yet uploaded. You have to use the flag `--image-repository {{ .REGISTRY }}`

Reply:
Fixed, PTAL.